MerchantSDK/GiniMerchantSDKExample/GiniMerchantSDKExample/Sources/DebugMenuViewController.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  DebugMenuViewController.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+
+protocol DebugMenuDelegate: AnyObject {
+    func didChangeReviewScreenSwitchValue(isOn: Bool)
+    func didChangeAmountEditableSwitchValue(isOn: Bool)
+}
+
+class DebugMenuViewController: UIViewController {
+    private let spacing = 20.0
+    private let rowHeight = 50.0
+
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.text = "Gini Merchant"
+        label.textAlignment = .center
+        label.font = .preferredFont(forTextStyle: .largeTitle)
+        return label
+    }()
+
+    private lazy var reviewScreenOptionLabel: UILabel = rowTitle("Show Review Screen")
+
+    private lazy var reviewScreenSwitch: UISwitch = {
+        let mySwitch = UISwitch()
+        mySwitch.translatesAutoresizingMaskIntoConstraints = false
+        mySwitch.addTarget(self, action: #selector(switchValueChanged(_:)), for: .valueChanged)
+        mySwitch.isOn = showReviewScreen
+        return mySwitch
+    }()
```

Comment:
maybe create func, that will build switch like rowTitle? for reviewScreenSwitch and amountEditableSwitch